### PR TITLE
Update port to 8080

### PR DESCRIPTION
**Pull request by WeihanLi**
update to `8080` since the default port changed to `8080`

https://learn.microsoft.com/en-us/dotnet/core/compatibility/containers/8.0/aspnet-port

it not working with current `80` port

image

works when updated to `8080`

image

image

### CODE REVIEW

**Comment by lbussell** [comment]
Hi @WeihanLi, thanks for your contribution. I pushed an update to your PR that updated lots of additional examples that include port 80. I'll try to get this merged soon.

File: samples/host-aspnetcore-https.md [hunk 1]
```diff
@@ -28,21 +28,26 @@ You need the [.NET 6 SDK or newer](https://dotnet.microsoft.com/download) for so
...
-> Note: `crypticpassword` is used as a stand-in for a password of your own choosing.
+> [!NOTE]
+> `crypticpassword` is used as a stand-in for a password of your own choosing.
 
 Run the container image with ASP.NET Core configured for HTTPS:
 
 ```console
 docker pull mcr.microsoft.com/dotnet/samples:aspnetapp
-docker run --rm -it -p 8000:80 -p 8001:443 -e ASPNETCORE_URLS="https://+;http://+" -e ASPNETCORE_HTTPS_PORT=8001 -e ASPNETCORE_Kestrel__Certificates__Default__Password="crypticpassword" -e ASPNETCORE_Kestrel__Certificates__Default__Path=/https/aspnetapp.pfx -v %USERPROFILE%\.aspnet\https:/https/ mcr.microsoft.com/dotnet/samples:aspnetapp
+docker run --rm -it -p 8000:8080 -p 8001:8001 -e ASPNETCORE_HTTPS_PORTS=8001 -e ASPNETCORE_Kestrel__Certificates__Default__Password="crypticpassword" -e ASPNETCORE_Kestrel__Certificates__Default__Path=/https/aspnetapp.pfx -v %USERPROFILE%\.aspnet\https:/https/ mcr.microsoft.com/dotnet/samples:aspnetapp
```
Comment (mthalman) [inline samples/host-aspnetcore-https.md:46]: Is `-p 8000:8080` even needed? The `8080` port won't be functional, right?

File: samples/run-aspnetcore-https-development.md [hunk 2]
```diff
@@ -88,10 +88,10 @@ docker build --pull -t aspnetapp .
 Run the container image with ASP.NET Core configured for HTTPS:
 
 ```console
-docker run --rm -it -p 8000:80 -p 8001:443 -e ASPNETCORE_URLS="https://+;http://+" -e ASPNETCORE_HTTPS_PORT=8001 -e ASPNETCORE_ENVIRONMENT=Development -v $env:APPDATA\microsoft\UserSecrets\:/root/.microsoft/usersecrets -v $env:USERPROFILE\.aspnet\https:/root/.aspnet/https/ aspnetapp
+docker run --rm -it -p 8001:8001 -e ASPNETCORE_HTTPS_PORTS=8001 -e ASPNETCORE_ENVIRONMENT=Development -v $env:APPDATA\microsoft\UserSecrets\:/root/.microsoft/usersecrets -v $env:USERPROFILE\.aspnet\https:/root/.aspnet/https/ aspnetapp
 ```
 
-After the application starts, navigate to `http://localhost:8000` in your web browser.
+After the application starts, navigate to `http://localhost:8001` in your web browser.
```
Comment (mthalman) [inline samples/run-aspnetcore-https-development.md:94]: Same change applies to other files as well.

```suggestion
After the application starts, navigate to `https://localhost:8001` in your web browser.
```

File: samples/run-in-sdk-container.md [hunk 3]
```diff
@@ -140,23 +140,24 @@ The following instructions demonstrate this scenario in various environments:
...
 ```
 
 ### Windows using Windows containers
 
 The following example uses PowerShell.
 
 ```console
-docker run --rm -it -p 8000:80 -v ${pwd}:C:\app -w C:\app mcr.microsoft.com/dotnet/sdk:7.0 dotnet run --launch-profile publicdev
+docker run --rm -it -p 8000:8080 -v ${pwd}:C:\app -w C:\app mcr.microsoft.com/dotnet/sdk:7.0 dotnet run --launch-profile publicdev
+
```
Comment (mthalman) [inline samples/run-in-sdk-container.md:160]: nit: Remove extra line

**Review by mthalman — APPROVED** [review]

**Pull request merged**